.github/actions/create-jira-issue/entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+sprint_id=$(curl -u ${JIRA_AUTH} -X GET -H "Content-Type: application/json" https://scalar-labs.atlassian.net/rest/agile/1.0/board/1/sprint?state=active | sed -n '/^{/,$p' | jq '.values[]' | jq '.id')
```

Comment:
Why do we need to use `sed`?
```suggestion
sprint_id=$(curl -u ${JIRA_AUTH} -X GET -H "Content-Type: application/json" https://scalar-labs.atlassian.net/rest/agile/1.0/board/1/sprint?state=active | jq -r '.values[].id')
```

Reply:
`sed` was needed to remove headers when I was using `-D-` with curl. Now it's not needed thank you.